CHANGELOG.md

```diff
@@ -2,14 +2,19 @@
 
 ## Master (Unreleased)
 
+## 3.0.0.pre (2024-06-05)
+
 - Remove extracted cops in `Capybara`, `FactoryBot` and `Rails` departments. ([@ydah])
 - Remove `RuboCop::RSpec::Language::NodePattern`. ([@ydah])
 - Remove `RSpec/FilePath` cop. ([@ydah])
-- Add new `RSpec/ExpectInLet` cop. ([@yasu551])
 - Remove `RSpec/Capybara/FeatureMethods` cop. If you are using this cop, change it to use `RSpec/Dialect`. ([@ydah])
 - Support `AutoCorrect: contextual` option for LSP. ([@ydah])
 - Enable all pending cops. ([@bquorning])
 
+## 2.30.0 (2024-06-03)
```

Comment:
master is behind 2.x-stable?

Reply:
The 2-x-stable branch isn’t an ancestor of master. But v2.30.0 exists on https://rubygems.org/gems/rubocop-rspec and I thought it might confuse users if the master branch changelog isn’t updated accordingly.

Keeping two branches is proving to be difficult.